docs/_articles/en/soql/soql-builder.md

```diff
@@ -1,13 +1,15 @@
 ---
-title: SOQL Builder (Beta)
+
+---
+title: SOQL Builder
 lang: en
 ---
 
-SOQL Builder is a VS Code extension that eliminates the guesswork when building SOQL queries. With SOQL Query Builder, anyone can visually build, run, and explore results from queries. Build queries using clicks in a visual editor, then save and extend the queries using a text editor. You can instantly view query results, and then save the results to a `.csv` or `.json` file.
```

Comment:
Is the official product name SOQL Query Builder? If so, I'd also change the title above.